### PR TITLE
Feature/style survey

### LINKED ISSUE #23: Style Preferences Component (8/25)
Create a style preferences component that exists on the user/style-preferences route and links to the store

### PR DESCRIPTION
**Pull request by Melizzo**
What's this PR do?
New Style Survey component was created and additional stylings were applied. 

Where should the reviewer start?
Starting api/web, and going to: http://localhost:3000/user/style-preferences

Any background context you want to provide?
What are the relevant tickets?
closes #23

Screenshots (if appropriate)
Screen Shot 2020-08-25 at 3 21 42 PM

### CODE REVIEW

File: code/web/src/modules/user/StyleSurvey.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      </GridCell>
+
+    <Grid>
+
+    </Grid>
+      </section>
+    );
+  }
+}
+
```
Comment (jpc20) [inline code/web/src/modules/user/StyleSurvey.js:58]: Nice, this is gonna look awesome!

**Review by jpc20 — APPROVED** [review]
Way to get the survey set up!

File: code/web/src/modules/user/StyleSurvey.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import { white, grey, grey2, grey3 } from "../../ui/common/colors";
+
+import { APP_URL } from "../../setup/config/env";
+import userRoutes from "../../setup/routes/user";
+
+
+class StyleSurvey extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```
Comment (Lithnotep) [inline code/web/src/modules/user/StyleSurvey.js:20]: I like  these style options!!

File: code/web/src/modules/user/StyleSurvey.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      counter: {
+        classy: 0,
+        artsy: 0,
+        punk: 0,
+        sporty: 0,
+        natureesque: 0
```
Comment (AntonioJacksonII) [inline code/web/src/modules/user/StyleSurvey.js:26]: Great choices for style options!

**Review by Lithnotep — APPROVED** [review]

**Review by garnetred — APPROVED** [review]

**Review by AntonioJacksonII — APPROVED** [review]
Great work! Thank you!

File: code/web/src/modules/user/StyleSurvey.js (hunk 3 above)
Comment (Melizzo) [reply code/web/src/modules/user/StyleSurvey.js:26]: December and I will add counter logic as the user select the choices of picture in the survey

**Pull request merged**